.github/workflows/ci.yml

```diff
@@ -5,96 +5,90 @@ on:
     branches:
       - main
     tags:
-      - '**'
+      - "**"
   pull_request: {}
 
 jobs:
   lint:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v4
+      - uses: actions/checkout@v4
 
-    - name: set up python
-      uses: actions/setup-python@v5
-      with:
-        python-version: '3.11'
+      - name: set up python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
 
-    - run: pip install -r requirements/linting.txt -r requirements/pyproject.txt pre-commit
+      - run: pip install -r requirements/linting.txt -r requirements/pyproject.txt pre-commit
 
-    - run: pre-commit run -a --verbose
-      env:
-        SKIP: no-commit-to-branch
+      - run: pre-commit run -a --verbose
+        env:
+          SKIP: no-commit-to-branch
 
   docs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v4
+      - uses: actions/checkout@v4
```

Comment:
does this need changing now (or at all)? It makes the other changes hear hard to see, and add unnecessary churn.

Reply:
Yep sorry about that, I'll get this this tuned back to just the salient changes